### DISCUSSION TITLE
Echelon EX3 to Zwift - Auto resistance and Gears

### DISCUSSION THREAD

**Discussion by marceastburn** [cagnulein/qdomyos-zwift#489] [Q&A]
Hello, I'm looking for some help with Auto resistance and gears using an Echelon ad Zwift. can anyone provide some example settings they have used as a starting point? 
Also, how are gears controlled when using a Raspberry PI as the bridge?

Thanks

**Comment by cagnulein** [comment]
Hi, first of all check #266
It's the start point for qml in case you're not using it ;)

**Reply by mrjcleaver** [reply]
ok, that answers Marc's "... Raspberry PI"... point.

What should I look at for the "Auto resistance and gears using an Echelon ad Zwift. can anyone provide some example settings"

I was looking for a wiki page or discussion - if something exists it's too buried.

Thanks, Martin

--
QZ on Huawei Android 7, Echelon EX5, QZ Companion on Samsung Galaxy Watch 7,  QZ AI on my phone (Samsung Galaxy 22 Ultra), Zwift on 4th gen Apple TV

**Reply by cagnulein** [reply]
Sorry @mrjcleaver why are you mentioning the raspberry pi? 

You would like to use your current setup with zwift? Just enable the gear tile in the tiles settings in qz and you are good to go!

Let me know

**Comment by mrjcleaver** [comment]
Hi Roberto,

I'm trying to understand how to use the Echelon EX 5 resistance changer to simulate using gears in Zwift.

![image](https://github.com/user-attachments/assets/b915f4f8-d782-4a80-ab41-b2cf2eb7dcda)

Thanks,
   Martin.

**Reply by cagnulein** [reply]
You have to enable the gears on knob setting under the echelon setting in qz under the bike options 
I have to warn you, it's experimental, so in case of issue create a ticket with a debug log https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F
